### ISSUE TITLE
navigation.wav

### ISSUE THREAD

**Issue by Itachi0833** [JosefNemec/Playnite#3597]
### Feature description

Hello, may I ask why the sound of navigation. wav will play when switching from desktop mode to full screen mode? Can this be removed? Thank you!

**Comment by JosefNemec** [comment]
Desktop mode doesn't play any audio at all. Must be some extension you have installed.

**Comment by JosefNemec** [comment]
If you mean selection sound in the actual Fullscreen mode after startup (completely unrelated to switch being done from Desktop mode), then that's because a game is being selected on startup.

**Comment by Itachi0833** [comment]
No, you don't understand what I mean. What I mean is that when you directly open full screen mode or switch from desktop mode to full screen mode, there will be a sound similar to Themes Fullscreen Default audio navigation. wav, just one sound. Every time you go to full screen mode, there will be a sound

**Comment by JosefNemec** [comment]
Then it's what I said in my second message, it's first game in the game list being selected. Not worth the effort changing how audio works just for startup, it's very minor "issue" in my opinion.

**Comment by Itachi0833** [comment]
I understand the principle by saying that. After thinking for a long time, I don't know what the problem is. Thank you
